src/ddmd/visitor.d

```diff
@@ -84,6 +84,11 @@ extern (C++) class Visitor
         visit(cast(Statement)s);
     }
 
+    void visit(ForwardingScopeStatement s)
```

Comment:
New members and C++ classes should be added to the headers to allow other backends to be aware of them, and so the vtables match.